charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml

```
@@ -0,0 +1,24 @@
+{{- if .Values.configs.clientserver.consoleDownload }}
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: enterprisecontract
+spec:
+  description: A copy of EnterpriseContract
```

Maybe something more user friendly here, e.g.:

```suggestion
  description: Enterprise Contract is a tool for verifying signatures and performing policy checks on container images